v2/support/basic_ticket_sample_app/assets/styles.css

```diff
@@ -0,0 +1,3 @@
+body {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
Copied from the basic web fallback that bootstrap uses: http://v4-alpha.getbootstrap.com/content/reboot/#native-font-stack